ui/redux/actions/sync.js

```diff
@@ -111,14 +111,14 @@ export function doSyncLoop(noInterval) {
     const syncEnabled = makeSelectClientSetting(SETTINGS.ENABLE_SYNC)(state);
     const syncLocked = selectSyncIsLocked(state);
     if (hasVerifiedEmail && syncEnabled && !syncLocked) {
-      dispatch(doGetSyncDesktop((error, hasNewData) => dispatch(doHandleSyncComplete(error, hasNewData))));
```

Comment:
We'll need to take a closer look at this doGetSyncDesktop function since it's supposed to use a password boolean, but that's getting removed now (broken even before your change).